cmd/kyma/alpha/create/module/module_test.go

```diff
@@ -150,7 +152,7 @@ func Test_command_getModuleTemplateAnnotations(t *testing.T) {
 				"annotation1": "value1",
 				"annotation2": "value2",
 				"operator.kyma-project.io/module-version": "1.1.1",
-				v1beta2.IsClusterScopedAnnotation:         v1beta2.DisableLabelValue,
+				shared.IsClusterScopedAnnotation:          v1beta2.DisableLabelValue,
```

Comment:
actually this`DisableLabelValue` and `EnableLabelValue` can be also moved to `shared`, but let's handle it in different issue, can you create a followup for it? Or maybe another PR in LM.

Reply:
Created a PR in lifecycle manager: https://github.com/kyma-project/lifecycle-manager/pull/1159